students/ChrisH/lesson03/list_lab.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+# -----------------------------------------------------------
+# list_lab.py
+#  demonstrates basic Python lists
+# -----------------------------------------------------------
+
+# Series 1
+print('*** Series 1')
+basket = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(basket)
+response = input("Please type the name of another fruit > ")
+basket.append(response)
+print(basket)
+response = int(input("Please type a number to display the corresponding fruit > "))
+if 0 < response <= len(basket):
+    print(response)
+    print(basket[response - 1])
+else:
+    print("No fruit at that index.")
+basket = ['Tangerine'] + basket
+print(basket)
+basket.insert(0, 'Kiwi')
+print(basket)
+for fruit in basket:
+    if fruit[0] == 'P':
+        print(fruit, " ", end="")
+print('')
+
+# Series 2
+print('\n*** Series 2')
+print(basket)
+basket.pop()
+print(basket)
+response = input("Enter a fruit from the list to delete > ")
+if response in basket:
+    basket.remove(response)
+else:
+    print("Did not find {:s}.".format(response))
```

Comment:
All wrap in `while` loop

Reply:
Added a While True loop to this section. (And one other section as well!)